tests/test_functions.py

```diff
@@ -11,20 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import json
+import os
 import pathlib
 import re
 import time
 
 import pretend
 import pytest
+from cloudevents.sdk import marshaller
+from cloudevents.sdk.converters import structured
+from cloudevents.sdk.event import v1
+from cloudevents.sdk.event import v03
+from cloudevents.sdk import converters
```

Comment:
I think it might be time to break this file up into:

* `test_http_functions.py`
* `test_event_functions.py`
* `test_cloudevent_functions.py`